framework/src/outputs/ConsoleUtils.C

```diff
@@ -51,7 +51,7 @@ outputFrameworkInformation(MooseApp & app)
 }
 
 std::string
-outputMeshInformation(FEProblemBase & problem, bool verbose)
+outputMeshInformation(const MooseApp & app, FEProblemBase & problem, bool verbose)
```

Comment:
Can't we just call `problem.getMooseApp()` instead of passing in the app separately?

Reply:
Yep...